cla-landing-page/src/app/components/cla-console-section/cla-console-section.component.html

```diff
@@ -91,4 +92,4 @@
             </div>
         </div>
     </div>
-</ng-template>
\ No newline at end of file
+</ng-template>
```

Comment:
@amolsontakke3576 Please add a new line at the end. I will approve it please make changes before merging.